src/test/e2e/tests/specs.test.ts

```diff
@@ -176,6 +180,56 @@ describe('Finding a court or tribunal listing', () => {
     });
   });
 
+  describe('Following the subscription \'search\' by case reference path', () => {
+    const validSearchTerm = '487065515';
+    const invalidSearchTerm = 'dddd';
+    const expectedNumOfResults = 1;
+
+    it('should open Session Mock Page to authenticate user', async () => {
+      await mockSessionPage.open('/mock-session');
+      expect(await mockSessionPage.getPageTitle()).toBe('Mock User Session Data');
+    });
+
+    it('should fill session form open subscription management page', async () => {
+      await mockSessionPage.enterText('Joe Bloggs', 'UsernameInput');
+      await mockSessionPage.enterText('1', 'UserIdInput');
+      await mockSessionPage.selectOption('UserType');
+      subscriptionManagementPage = await mockSessionPage.clickContinue();
+      expect(await subscriptionManagementPage.getPageTitle()).toBe('Your subscriptions');
+    });
+
+    it('should open the subscription add page', async () => {
+      await subscriptionAddPage.open('subscription-add');
+      expect(await subscriptionAddPage.getPageTitle()).toBe('How do you want to add a subscription?');
+    });
+
```

Comment:
we already have all these steps, you should be branching at the appropriate split points in the journey rather tan duplicating full journeys

Reply:
fixed.